Bug fixes.

GoldenWRaft opened the pull request:
1. Config file not updating while opening and closing files.
2. Fixed open file location from static to %USERPROFILE%/Desktop (Only working for windows)
3. When a name is changed unnecessary updates mitigated

GoldenWRaft committed:
Bug fixes.

GoldenWRaft committed:
Offline Version

GoldenWRaft commented:
> 1. Config file not updating while opening and closing files.
> 2. Fixed open file location from static to %USERPROFILE%/Desktop (Only working for windows)
> 3. When a name is changed unnecessary updates mitigated
> 
> 

@face-hh

GoldenWRaft committed:
Update main.rs

Co-authored-by: Agatem <agaatem@outlook.com>

agatemosu commented on src-tauri/src/main.rs:
```diff
@@ -72,15 +71,18 @@ fn save_config() {
+        });
 
-    println!("Removed file: {:?}", path);
+        return 0;
```
didn't see sorry

```suggestion
        return 0;
    }
```

agatemosu commented on src-tauri/src/main.rs:
```diff
@@ -27,8 +27,7 @@ struct Child {
 lazy_static! {
-    static ref PATH: Mutex<String> =
-        Mutex::new("C:/Users/User/Documents/GitHub/text-editor/Folder".to_string());
+    static ref PATH: Mutex<String> = Mutex::new("%USERPROFILE%/Desktop".to_string());
```
idk what this does but in other languages the user folders is not in english (at least on linux)

GoldenWRaft committed:
Bug Fixes

GoldenWRaft committed:
Merge branch 'main' of https://github.com/GoldenWRaft/TextEditorInRust

GoldenWRaft committed:
Bug Fixes

GoldenWRaft committed:
Line navigation issue fixed.

Converted options into a programmably generated list instead of hard coded HTML

GoldenWRaft committed:
Word count corrected

GoldenWRaft committed:
Update CI.yml

GoldenWRaft committed:
Update CI.yml

Build script fixes

GoldenWRaft committed:
Test Changes

GoldenWRaft committed:
Cargo Check

GoldenWRaft committed:
Failed Build updated

GoldenWRaft committed:
Update CI.yml

GoldenWRaft committed:
Update CI.yml

GoldenWRaft committed:
Update CI.yml

GoldenWRaft committed:
Update CI.yml

GoldenWRaft committed:
Updates

GoldenWRaft committed:
Update CI.yml

GoldenWRaft committed:
Update CI.yml

lockedmutex commented:
@GoldenWRaft checkout [Rhyolite](https://github.com/RedddFoxxyy/Rhyolite)
